src/platform/imx8/lib/dma.c

```diff
@@ -31,7 +31,7 @@ struct dma dma[PLATFORM_NUM_DMACS] = {
 		.id		= DMA_ID_HOST,
```

Comment:
Commit message should be more like this: "This is to optimize memory consumption and avoid allocation a larger memory block (>1024 bytes). 16 Host DMA channels should be enough for everyone"